src/command/FileCommands.cxx

```diff
@@ -234,9 +236,93 @@ handle_read_comments(Client &client, Request args, Response &r)
 	gcc_unreachable();
 }
 
+AllocatedPath
+find_art(AllocatedPath directory)
+{
+    std::vector<std::string> art_names = {
```

Comment:
We can always make that dynamic/configurable later; and once we do that, we'll have one global `std::vector` instead of allocating a new one for each function call.